src/getConfig.js

```diff
@@ -127,18 +127,14 @@ function validateConfig(config) {
   Please remove ${chalk.bold('gitDir')} from your configuration.`
   }
 
-  const validation = validate(config, {
+  validate(config, {
```

Comment:
Let's add a comment here explaining why we do not need to check the object which is returned from calling `validate`.

Reply:
Is it necessary? Personally, I consider the `validate` function to be self-explaining